docs/team/jrmckh.md

```diff
@@ -0,0 +1,51 @@
+---
+Chia Jeremy's Project Portfolio Page
+---
+
+### <span style="color:orange"> Project: EzSchedule </span>
+
+EZ-Schedule is a desktop application used for managing and scheduling of events. The user interacts with it using a
+Command Line Interface (CLI), and it has a Graphical User Interface (GUI) created with JavaFX. It is written in Java,
+and has about ***to be added soon*** kLoC.
+
+Given below are my contributions to the project.
+
+### Overview
+* [Summary of Contributions](#summary-of-contributions-)
+* [Code contributed](#code-contributed-)
+* [New Features](#new-features-)
+* [Enhancements to existing features](#enhancements-to-existing-features-)
+* [Project management](#project-management-)
+* [Documentation](#documentation-)
+* [Community](#community-)
+* [Tools](#tools-)
```

Comment:
Not too sure if the links need the tailing dash at the back.
Eg. Im not too sure if its `#tools-` or `#tools`